string process_id
float32 max_duration
---
int32 nb_holders
